sympy/assumptions/handlers/calculus.py

```diff
@@ -52,7 +63,7 @@ def Symbol(expr, assumptions):
         return None
 
     @staticmethod
-    def Add(expr, assumptions):
+    def Add(expr, *, assumptions):
```

Comment:
Changing assumptions to be keyword-only is making this diff huge, and as far as I can tell is orthogonal. For the sake of making review easier, can you either:

* Make a PR that does nothing other than make assumptions keyword-only, and rebase this PR once that gets merged
* Remove these changes from this PR, and make the keyword-only arguments a follow-up

Thanks!